### DISCUSSION TITLE
New tab opened UI feedback on compact mode

### DISCUSSION THREAD

**Discussion by migtorr** [zen-browser/desktop#974] [Ideas]
I like opening new tabs by doing middle click on links but I don't see any visual indication the tab was opened in the background. 

I would like to suggest like a glow or the tab square making a quick one time jump from the sidebar when a new tab is opened on the background while on compact mode.

**Comment by elia169** [comment]
Same happens when clicking "open tab in new window", would agree that it should also show the sidebar in compact mode, probably an oversight
